rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-extractfrompdf.adoc

```diff
@@ -0,0 +1,64 @@
+# Finding Information in a PDF
```

Comment:
Is this formatted this way for H&T?

Reply:
This is a new type of documentation inside MuleSoft help. Diane suggested to use it as a prototype.
Users/SEs find it difficult to assemble features of different RPA products to solve their use case. 
We wanted to include possible search terms in the texts and headings, so readers can find it easily. The headings reflect the kind of questions users put previously in our Slack channnels, e.g. 'How do you find a specific information in a PDF?', and probably use as serach terms in our docs.

Reply:
And, yes, you're right, the idea was to transform this topic into a training at a later time.

Reply:
So this (hash sign) works in the local build for headings?

Reply:
Oh - I'm so sorry! This is Markdown syntax, which accidentally works in the local build. I will change this.